e2e_test/test/e2e_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('setup test', () {
+    test('dazel init', () {
+      expectSuccess(Process.runSync('pub', ['run', 'dazel', 'init']));
+
+      bazel(['version']);
+      bazel(['clean']);
+    });
+  });
+
+  group('e2e test', () {
+    setUp(() {
```

Comment:
Also might-could be `setUpAll` assuming we don't want to re-init every time...right?